src/easydmp/dmpt/admin.py

```diff
@@ -132,6 +144,79 @@ class TemplateAdmin(SetPermissionsMixin, ObjectPermissionModelAdmin):
         }),
     )
 
+    # extra buttons on changelist
+
+    def import_template(self, request):
+        if not request.user.has_superpowers:
+            raise PermissionDenied
+        request.current_app = self.admin_site.name
+        opts = self.model._meta
+        if request.method == 'POST':
+            template_export = request.FILES['template_export_file']
+            form = TemplateImportForm(request.POST, request.FILES)
+            if form.is_valid():
+                template_export_file = request.FILES['template_export_file']
+                template_export_jsonblob = template_export_file.read()
+                url_on_error = reverse('%s:%s-%s-import' % (
+                                        self.admin_site.name,
+                                        opts.app_label, opts.model_name))
+                try:
+                    serialized_dict = deserialize_template_export(template_export_jsonblob)
+                except TemplateImportError as e:
+                    error_msg = f'{e}, cannot import'
+                    messages.error(request, error_msg)
+                    return HttpResponseRedirect(url_on_error)
+
+                try:
+                    with warnings.catch_warnings(record=True) as w:
+                        tim = import_serialized_export(serialized_dict, via='admin')
```

Comment:
It's a little bit nicer to either use an enum for these, or define consant for also 'admin', so it's less "hidden away".

Reply:
I have a branch for importing via API (and via fetching from an url), so I'll be doing something smarter then, I think.